ExcerciseNo1.java

```diff
@@ -0,0 +1,33 @@
+package com.journaldev.readfileslinebyline;
+
+import java.io.*;
+import java.util.*;
+
+public class ExcerciseNo1 {
+    public static void main(String[] args) throws FileNotFoundException {
+        File file = new File("txtfile.txt");
+        Scanner scan = new Scanner(file);
+        String data;
+        ArrayList<String> words = new ArrayList<String>();
+        while (scan.hasNextLine()) {
+            data = scan.nextLine().toLowerCase();
```

Comment:
where i say in excersie about changing case?

Reply:
Fixed